Langref polish (#2193)
slint-ui/slint: docs/langref/src/properties.md

}
```

> In addition to the existing properties, you may define extra properties. In
> additon to the property name you must also specify the type of the property
> and optionally a default value:

Reviewer: ```suggestion
In addition to the existing properties, define extra properties by specifying the
name, the type, and optionally a default value:
```

}
```

> You can annotate custom the properties with a qualifier that specifies how the
> property can be read and written:

Reviewer: ```suggestion
Annotate custom the properties with a qualifier that specifies how the
property can be read and written:
```
